scripts/tutorial-uploader/README.md

```diff
@@ -0,0 +1,40 @@
+# Tutorial uploader
+
+This script uploads tutorials to IBM Quantum Learning.
+
+## Developing
+
+To work on this script, you'll need to set up the `saiba-api` project locally.
+
+There are some extra steps you'll need to take to set up `saiba-api` for
+developing this script:
+
+- Follow instruction in the README in the saiba-api repo. Don't forget the
```

Comment:
Nit: Maybe I'm wrong here and this is correct, but just in case :)

```suggestion
- Follow the instructions in the README in the saiba-api repo. Don't forget the
```

Reply:
No you're completely right, thanks!